### DISCUSSION TITLE
Use GitHub projects to better organize this board ;)

### DISCUSSION THREAD

**Discussion by pedro-santiago** [laravel/nova-issues#5258] [General]
It's way easier to check roadmap, other developers ideas and check out what's planned. It will allow people to be more calm knowing the direction nova is taking up ahead.

**Comment by bkuhl** [comment]
I agree.  I find this Discussions board to be of limited usefulness.  There's not even a way to filter items by most upvoted, so I can't imagine they're able to focus on the most upvoted items unless people are actively searching the backlog of Discussions and bumping them to the front page.
